### PR TITLE
feat: add version parameter to allow version override

### PR DESCRIPTION
**Pull request by M3lkior**
See asyncapi/html-template#107

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ npm install -g @asyncapi/generator
 |Name|Description|Required|Default|Example|
 |---|---|---|---|---|
 |outFilename|The filename of the output file.|No|`asyncapi.md`|`index.md`|
+|version|Override the Async API version displayed in the generated pages.|No|*Any* ([See Semver versionning](https://semver.org/))|`1.0.0`|
```
Comment (derberg) [inline README.md:23]:
```suggestion
| version | Override the version of your application provided under `info.version` location in the spec file. | No | Version is taken from the spec file. |`1.0.0`|
```

File: package.json [hunk 2]
```diff
@@ -70,6 +70,10 @@
         "description": "The name of the output markdown file",
         "default": "asyncapi.md",
         "required": false
+      },
+      "version": {
+        "description": "Override the Async API version displayed in the generated pages.",
```
Comment (derberg) [inline package.json:75]:
```suggestion
        "description": "Override the version of your application provided under `info.version` location in the spec file.",
```

**Review by derberg — COMMENTED** [review]
@M3lkior thanks a lot for this PR. Please improve those descriptions.

File: README.md (hunk 1 above)
Comment (M3lkior) [reply README.md:23]: hi, is it normal that you suggest adding `Version is taken from the spec file.` into the `Allowed values` ?

Comment (derberg) [reply README.md:23]: what `Allowed values` do you mean? I suggest it under `Default`

Comment (M3lkior) [reply README.md:23]: in `html-template`, this is a `Allowed values` column

File: package.json (hunk 2 above)
Comment (M3lkior) [reply package.json]: done

File: README.md (hunk 1 above)
Comment (M3lkior) [reply README.md]: done for the `markdown` side

Comment (derberg) [reply README.md]: ah yes, we are not consistent yet, we set some standard here https://github.com/asyncapi/template-for-generator-templates/blob/master/_README.md#template-configuration just need to adjust, could you do this in this and the other PR? 🙏🏼

**Review by derberg — APPROVED** [review]
@M3lkior thanks a lot 🙏🏼

File: README.md (hunk 1 above)
Comment (M3lkior) [reply README.md]: yes, it is done

**Comment by derberg** [comment]
@all-contributors please add @M3lkior for code

**Pull request merged**